client/src/app/features/member-features/member-detail/pages/member-detail-page/member-detail-page.component.ts

```
@@ -38,24 +39,32 @@ export class MemberDetailPageComponent implements OnInit, OnDestroy {
-        this.member &&
-          this.member.photos.map((p) => {
-            this.images.push(new ImageItem({ src: p.url, thumb: p.url }));
-          });
+    combineLatest([
+      this.route.data.pipe(
+        tap((data) => {
+          this.member = data['member'];
+          this.member &&
+            this.member.photos.map((p) => {
+              this.images.push(new ImageItem({ src: p.url, thumb: p.url }));
+            });
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)